File: c programming/assigments/module 3.2/Q9.c
#include <stdio.h>

		//Write a program to find out the max from given number (E.g., No: -1562Max number is 6). 

int main()
{
    int num1, num2;

   
    printf("Enter two numbers: ");
    scanf("%d%d", &num1, &num2);

    
    if(num1 > num2)
    {
        printf("%d is maximum", num1);        
    }

   
    if(num2 > num1)
    {
        printf("%d is maximum", num2);
    }

   
    if(num1 == num2)
    {
        printf("Both are equal");
    }

    return 0;
}

File: fundamentals of python/q9.py
#Write a Python program to sum of three given integers. However, if two values are equal sum will be zero.



def sum_three_unique_numbers(a, b, c):
    if a == b or b == c or a == c:
        return 0
    else:
        return a + b + c


num1 = int(input("Enter the first number: "))
num2 = int(input("Enter the second number: "))
num3 = int(input("Enter the third number: "))

result = sum_three_unique_numbers(num1, num2, num3)
print("The sum of the three numbers is:", result)